third_party/bazel_embedded/repos.bzl

```diff
@@ -4,12 +4,18 @@
 
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
-def bazel_embedded_repos():
+def bazel_embedded_repos(local=None):
     # Contains rules that support building SW for embedded targets. Specifically, we
     # maintain a fork to build for RISCV32I.
-    git_repository(
-        name = "bazel_embedded",
-        commit = "fe65a8aca35ae0bae563efd6a29cc14fcb15140d",
-        remote = "https://github.com/lowRISC/bazel-embedded.git",
-        shallow_since = "1639417565 -0800",
-    )
+    if local:
+        native.local_repository(
+            name = "bazel_embedded",
+            path = local,
+        )
+    else:
+        git_repository(
```

Comment:
#11996 changes this to be a `http_archive`, so there could be a conflict depending on what gets merged first.

Reply:
#11996 merged